Packs/Base/ReleaseNotes/1_31_29.md

```diff
@@ -0,0 +1,4 @@
+#### Scripts
+##### SanePdfReports
+- Updated the Docker image to: *demisto/sane-pdf-reports:1.0.0.36824*.
+  - Fix bug that panels were overlapping on each other
```

Comment:
```suggestion
  - Fixed an issue where panels would overlap each other.
```

Reply:
@amshamah419 thanks.
I also want to add 'Fixed an issue where the report date was missing'
is it ok?

Reply:
That sounds alright.